styx-e2e-test/flyte-test-workflow/workflows/hello_world_workflow.py

```diff
@@ -0,0 +1,7 @@
+from flytekit.sdk.workflow import workflow_class
+from workflows.hello_world_task import print_hello_world
+
+
+@workflow_class()
```

Comment:
How about we disable the default launch plan and create one explicitly? So that we can set KSA instead of using `assumable_iam_role:arn:aws:iam::123:role/test-role`.